### PR TITLE
Better Organize the database structure

### PR DESCRIPTION
**Pull request by michael-beaver**
This PR changes the structure of the DB folder to better handle passing the knex connection around and having the database functions split up into different files

### CODE REVIEW

File: db/connect.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict'
```
Comment (renodubois) [inline db/connect.js:1]: Didn't we agree that we didn't need these anymore w/ ES6?

**Review by renodubois — COMMENTED** [review]
Just this one comment, but overall looks good! Makes a lot more sense this way.

File: db/connect.js (hunk 1 above)
Comment (michael-beaver) [reply db/connect.js:1]: I'm not 100% on if we should get rid of it or not, but if we do, we should get rid of all of them at the same time in a separate PR. So for now I just want to be consistent with the rest of the project

Comment (renodubois) [reply db/connect.js:1]: Sounds good, we can handle that later.

**Review by renodubois — APPROVED** [review]

**Pull request merged**